tests/unittests/BackendTest.cpp

```diff
@@ -53,19 +55,24 @@ TEST(Interpreter, profileQuantizationForANetwork) {
   Node *O = F->createFullyConnected(bindings, "fc", A, 4);
   O = F->createRELU("relu", O);
   O = F->createRegression("reg", O, Ex);
+  F->createSave("ret", O);
 
   LoweredInfoMap loweredMap;
   CompilationContext cctx{&bindings, &loweredMap};
   cctx.precisionConfig.quantMode = QuantizationMode::Profile;
 
   bindings.allocate(A);
   bindings.allocate(Ex);
-  EE.compile(F, cctx);
+  EE.compile(cctx);
+  bindings.allocate(mod.getPlaceholders());
 
   // TODO: Verify histogram itself, for now just verify min and max.
   // Run inference first time and capture tensor stats.
-  updateInputPlaceholders(bindings, {A}, {&inputs});
+  updateInputPlaceholders2(bindings, {A}, {&inputs});
   EE.run(bindings);
+  // The partitioner deleted the original function and created a new one, get
+  // the new function.
```

Comment:
very nit: I think the comment could be more accurate : " The partitioner **may** delete the original function " :)  Since it is possible that there is no partition and no function deletion.